test/zkproof/flows_test.go

```diff
@@ -1047,3 +1047,152 @@ func TestZKInputsAtomicTxsEdge(t *testing.T) {
 	assert.Equal(t, expectedRoots[6], bb.LocalStateDB().MT.Root().BigInt().String())
 	sendProofAndCheckResp(t, zki)
 }
+
```

Comment:
I thought we had an agreement for not using TIL anymore. It creates problem for us when a block is invalid in blockchain and today the Synchroniser is returning nil to allow TIL to work (vide details here https://github.com/hermeznetwork/hermez-node/issues/899 and here https://github.com/hermeznetwork/hermez-node/pull/900 ).
We cannot remove TIL for now but for new tests please use ```geth --dev``` or use the package https://github.com/ethereum/go-ethereum/tree/master/accounts/abi/bind/backends of GETH that creates a blockchain on the fly and you can perform and control block generation within the tests.

Reply:
After a meet, we've decided to think about this later on